### PR TITLE
new package: cantata media player

### PR DESCRIPTION
**Pull request by Yisus7u7**
Screenshot_20220321-105519

Cantata is an interface for MPD, supports local music playback, stations (FM), online music and podcast, it is also a client for Apple Music.

### CODE REVIEW

**Comment by mguinhos** [comment]
Looks a gorgeous software!

**Comment by Yisus7u7** [comment]
Done.

**Comment by xtkoba** [comment]
I see some potential hard-coded paths in `bin/cantata`:

```
/etc/fstab
/etc/mpd.conf
```

Please prepend `@TERMUX_PREFIX@` to each of them.

**Comment by Yisus7u7** [comment]
> I see some potential hard-coded paths in `bin/cantata`:
> 
> ```
> /etc/fstab
> /etc/mpd.conf
> ```
> 
> Please prepend `@TERMUX_PREFIX@` to each of them.

Where is that file located?

**Comment by Yisus7u7** [comment]
@xtkoba Done.

File: packages/cantata/use_termux_path.patch [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+--- a/playlists/cantata-dynamic.service
++++ b/playlists/cantata-dynamic.service
+@@ -7,8 +7,8 @@ After=mpd.service
+ User=mpd
+ Group=audio
+ Type=forking
+-ExecStart=/usr/share/cantata/scripts/cantata-dynamic server /etc/cantata-dynamic.conf
+-ExecStop=/usr/share/cantata/scripts/cantata-dynamic stopserver /etc/cantata-dynamic.conf
++ExecStart=/data/data/com.termux/files/usr/share/cantata/scripts/cantata-dynamic server /data/data/com.termux/files/usr/etc/cantata-dynamic.conf
++ExecStop=/data/data/com.termux/files/usr/share/cantata/scripts/cantata-dynamic stopserver /data/data/com.termux/files/usr/etc/cantata-dynamic.conf
```
Comment (xtkoba) [inline packages/cantata/use_termux_path.patch:106]: In a patch you can use `@TERMUX_PREFIX@` instead of `/data/data/com.termux/files/usr`.

**Pull request closed without merging**